Jenkinsfile

```diff
@@ -291,6 +291,11 @@ pipeline {
                                 withEnv(['PATH+TBB=./lib/tbb']) {
                                     sh "python ./test/expressions/test_expression_testing_framework.py"
                                 }
+                                sh "make clean-all"
+                                sh "echo STAN_THREADS=true >> make/local"
+                                withEnv(['PATH+TBB=./lib/tbb']) {
+                                    sh "./runTests.py -j${env.PARALLEL} test/expressions"
```

Comment:
I see the tests above are wrapped in a try/finally. Something about a junit. Does that need to happen here?

Reply:
Fixed. Thanks!